### PR TITLE
Header component with project switch drawer

### PR DESCRIPTION
**Pull request by braj176**
Header component with project switch drawer

### CODE REVIEW

File: vizdom/src/core/routes/routes.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ const Routes = [
     path: "/mob-bottom-menu",
     page: BottomMenu,
   },
+  {
+    path: "/header",
+    page: HeaderComponent,
+  },
```
Comment (PrashantHalaki) [inline vizdom/src/core/routes/routes.js:42]: Why do you need a separate route for the header? Integrate it with pages

**Review by PrashantHalaki — CHANGES_REQUESTED** [review]
@braj176 Remove the routes for the components. Routes should only be there for pages.

File: vizdom/src/core/routes/routes.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import InputFormats from "../../pages/inputFormats";
 import Components from "../../components/AllComponents";
 import SideMenu from "../../components/webSidebarMenu";
 import BottomMenu from "../../components/mobBottomMenu";
-
+import HeaderComponent from "../../components/header";
```
Comment (PrashantHalaki) [inline vizdom/src/core/routes/routes.js:8]: Remove unused import @braj176

Comment (braj176) [reply vizdom/src/core/routes/routes.js]: removed unused import

**Pull request merged**